raiden/transfer/node.py

```diff
@@ -395,6 +397,41 @@ def sanity_check(iteration: TransitionResult):
     assert isinstance(iteration.new_state, ChainState)
 
 
+def inplace_delete_message_queue(
+        chain_state: ChainState,
+        state_change: StateChange,
+        queueid: QueueIdentifier,
+):
+    """ Filter messages from queue, if the queue becomes empty, cleanup the queue itself. """
+    queue = chain_state.queueids_to_queues.get(queueid)
+    if not queue:
+        return
+
+    filtered_queue = inplace_delete_message(
```

Comment:
`inplace_delete_message` doesn't return, `filtered_queue` will always be none and this means the queue will always be cleared.